packages/e2e/test/local-network/local-network.test.ts

```diff
@@ -108,4 +109,190 @@ describe('Local Network', () => {
     wallet1.shutdown();
     wallet2.shutdown();
   });
+
+  it('will mint an NFT using native scripts.', async () => {
```

Comment:
Would be best to make this test network-agnostic. Shouldn't be too hard?

Reply:
This was particular test was removed, the nft test we already have will cover this uses cases, that test was just made network agnostic :)